docs/source/developerguide.rst

```diff
@@ -100,9 +105,12 @@ the model version (``-1`` referring to the latest), and the inference server log
 
    infer_server_config = NVInferServer.generate_default_config()
    infer_server_config.infer_config.backend.trt_is.model_repo.root = "/app/models"
-   infer_server_config.infer_config.backend.trt_is.model_name = "cholec_unet_864x480"
+   infer_server_config.infer_config.backend.trt_is.model_name = "us_unet_256x256"
```

Comment:
Shouldn't this be monai_unet_trt?

Reply:
no, it should match the folder name under `app/models`

Reply:
👍